integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/mutation/CarbonProjectForDeleteCommand.scala

```diff
@@ -104,24 +111,27 @@ private[sql] case class CarbonProjectForDeleteCommand(
       // handle the clean up of IUD.
       CarbonUpdateUtil.cleanUpDeltaFiles(carbonTable, false)
 
-      val deletedSegments = DeleteExecution.deleteDeltaExecution(
+      val (deletedSegments, deletedRowCountTemp) = DeleteExecution.deleteDeltaExecution(
```

Comment:
why not assign to deletedRowCount directly?

Reply:
Done